src/libraries/System.Runtime/tests/System/DoubleTests.cs

```diff
@@ -272,6 +272,11 @@ public static IEnumerable<object[]> Parse_Valid_TestData()
             yield return new object[] { (567.89).ToString(), defaultStyle, null, 567.89 };
             yield return new object[] { (-567.89).ToString(), defaultStyle, null, -567.89 };
             yield return new object[] { "1E23", defaultStyle, null, 1E23 };
+            yield return new object[] { "9007199254740997.0", defaultStyle, invariantFormat, 9007199254740996 };
```

Comment:
```suggestion
            yield return new object[] { "9007199254740997.0", defaultStyle, invariantFormat, 9007199254740996.0 };
```

To ensure the result is explicitly `double` and not a `long`

Reply:
The corresponding unit test parameter is typed as a double, so it shouldn't matter. I changed it nevertheless though.